en/core-api/e-wallet.md

```diff
@@ -1,76 +1,75 @@
 # Core API E-Wallet Integration
 <hr>
 
-?> This payment method is compatible with [QR Code Indonesian Standard (QRIS)](https://www.bi.go.id/id/ruang-media/siaran-pers/Pages/SP_216219.aspx), and can be paid with **any QRIS compatible e-wallet or banking app**.
+Midtrans facilitates its customers by *E-Wallet*. This payment method is compatible with [QR Code Indonesian Standard (QRIS)](https://www.bi.go.id/id/ruang-media/siaran-pers/Pages/SP_216219.aspx). Customers can make payments with GoPay, ShopeePay or any QRIS compatible e-wallet or banking app. 
 
-GoPay is an e-Wallet payment method by Gojek. Users will pay using the Gojek apps, or any QRIS compatible app. The user flow varies when using a web browser (on a computer or a tablet) compared to a SmartPhone:
+Gojek has introduced GoPay as an E-Wallet payment method. Customers can pay using the Gojek apps, or any QRIS compatible app.  The user flow is different for web browser (on a computer or a tablet) and smartphone. The user flows are given below.
 
-1. **QR Code** - This is the user flow on a web browser (on a computer or a tablet). User is shown a QR code and asked to scan using any QRIS compatible app, like Gojek app.
-2. **Deeplink** - This is the user flow on a SmartPhone/mobile device. User gets redirected to the Gojek apps to finish payment.
+1. **QR Code** - This is the user flow on a web browser (on a computer or a tablet). User is shown a QR code and asked to scan using any QRIS compatible app, such as Gojek app.
+2. **Deeplink** - This is the user flow on a smartphone /mobile device. User gets redirected to the Gojek app to finish payment.
 
-Basic integration process of GoPay will be explained below.
-
-?> Please make sure you have already done [creating your Midtrans Account](/en/midtrans_account/overview.md), before proceeding with this section.
-
-## Integration Step
-1. Send transaction data to API Charge.
-2. Alter payment flow depending on the device used.
-3. Handling Post-Transaction.
```

Comment:
Please refer to this feedback https://github.com/Midtrans/technical-documentation-site/pull/24#pullrequestreview-522269444
Plus, most of the feedback there also applicable here.